### PR TITLE
chore: Remove UI packages

### PR DESCRIPTION
**Pull request by ericclemmons**
#### Description of changes

This PR removes UI-packages (**with the exception of `aws-amplify-react-native`) and directs customers to https://github.com/aws-amplify/amplify-ui/tree/legacy/legacy.

With the creation of https://github.com/aws-amplify/amplify-ui and launch of https://ui.docs.amplify.aws/, Amplify UI is a 1st-class citizen with a dedicate team.

So the ongoing maintenance (and most importantly, **release**) of UI packages will be managed by the @aws-amplify/amplify-ui team.

This PR keeps a `README.md` in each folder for any customers or developers who aren't aware of the UI efforts, similar to:
> https://github.com/amazon-archives/amazon-cognito-identity-js

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#steps-towards-contributions)
- [x] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by sammartinez — CHANGES_REQUESTED** [review]
We should have a convo on this one Eric as this will be a breaking change and will require a major version bump which we will not being doing until next year sometime

**Comment by ericclemmons** [comment]
@sammartinez Got it!  Let's coordinate with @Milan-Shah on our short-term needs and find an alternate solution.

File: packages/aws-amplify/__tests__/exports-test.ts [hunk 1]
```diff
@@ -25,63 +25,6 @@ describe('aws-amplify', () => {
 			  "PubSub",
 			  "Cache",
 			  "Interactions",
-			  "a",
```
Comment (ericclemmons) [inline packages/aws-amplify/__tests__/exports-test.ts:28]: These snapshots were the generated CSS classnames:

```js
  a: 'Anchor__a___1_Iz8',
  button: 'Button__button___vS7Mv',
  signInButton: 'Button__signInButton___3bUH-',
  googleSignInButton: 'Button__googleSignInButton___1YiCu',
  signInButtonIcon: 'Button__signInButtonIcon___ihN75',
  auth0SignInButton: 'Button__auth0SignInButton___znnCj',
  facebookSignInButton: 'Button__facebookSignInButton___34Txh',
  amazonSignInButton: 'Button__amazonSignInButton___2EMtl',
  oAuthSignInButton: 'Button__oAuthSignInButton___3UGOl',
  signInButtonContent: 'Button__signInButtonContent___xqTXJ',
  formContainer: 'Form__formContainer___1GA3x',
  formSection: 'Form__formSection___1PPvW',
  formField: 'Form__formField___38Ikl',
  formRow: 'Form__formRow___2mwRs',
  hint: 'Hint__hint___2XngB',
  input: 'Input__input___3e_bf',
  inputLabel: 'Input__inputLabel___3VF0S',
  label: 'Input__label___23sO8',
  radio: 'Input__radio___2hllK',
  navBar: 'Nav__navBar___xtCFA',
  navRight: 'Nav__navRight___1QG2J',
  nav: 'Nav__nav___2Dx2Y',
  navItem: 'Nav__navItem___1LtFQ',
  photoPickerButton: 'PhotoPicker__photoPickerButton___2XdVn',
  photoPlaceholder: 'PhotoPicker__photoPlaceholder___2JXO4',
  photoPlaceholderIcon: 'PhotoPicker__photoPlaceholderIcon___3Et71',
  container: 'Section__container___3YYTG',
  actionRow: 'Section__actionRow___2LWSU',
  sectionHeader: 'Section__sectionHeader___2djyg',
  sectionHeaderHint: 'Section__sectionHeaderHint___3Wxdc',
  sectionBody: 'Section__sectionBody___ihqqd',
  sectionHeaderContent: 'Section__sectionHeaderContent___1UCqa',
  sectionFooter: 'Section__sectionFooter___1T54C',
  sectionFooterPrimaryContent: 'Section__sectionFooterPrimaryContent___2r9ZX',
  sectionFooterSecondaryContent: 'Section__sectionFooterSecondaryContent___Nj41Q',
  selectInput: 'SelectInput__selectInput___3efO4',
  strike: 'Strike__strike___1XV1b',
  strikeContent: 'Strike__strikeContent___10gLb',
  toast: 'Toast__toast___XXr3v',
  toastClose: 'Toast__toastClose___18lU4',
  totpQrcode: 'Totp__totpQrcode___1crLx',
  sumerianSceneContainer: 'XR__sumerianSceneContainer___3nVMt',
  sumerianScene: 'XR__sumerianScene___2Tt7-',
  loadingOverlay: 'XR__loadingOverlay___IbqcI',
  loadingContainer: 'XR__loadingContainer___2Itxb',
  loadingLogo: 'XR__loadingLogo___Ub7xQ',
  loadingSceneName: 'XR__loadingSceneName___3__ne',
  loadingBar: 'XR__loadingBar___2vcke',
  loadingBarFill: 'XR__loadingBarFill___3M-D9',
  sceneErrorText: 'XR__sceneErrorText___2y0tp',
  sceneBar: 'XR__sceneBar___2ShrP',
  sceneName: 'XR__sceneName___1ApHr',
  sceneActions: 'XR__sceneActions___7plGs',
  actionButton: 'XR__actionButton___2poIM',
  tooltip: 'XR__tooltip___UYyhn',
  actionIcon: 'XR__actionIcon___2qnd2',
  autoShowTooltip: 'XR__autoShowTooltip___V1QH7',
```

**Comment by wlee221** [comment]
There are some new commits since we migrated to [aws-amplify/amplify-ui](https://github.com/aws-amplify/amplify-ui):

- https://github.com/aws-amplify/amplify-js/pull/8048
- https://github.com/aws-amplify/amplify-js/pull/8832
- https://github.com/aws-amplify/amplify-js/pull/8910 (no actual code changes)
- https://github.com/aws-amplify/amplify-js/pull/8953
- https://github.com/aws-amplify/amplify-js/pull/8993
- https://github.com/aws-amplify/amplify-js/pull/8984
- https://github.com/aws-amplify/amplify-js/pull/8994 (no actual code changes)
- https://github.com/aws-amplify/amplify-js/pull/9021
- https://github.com/aws-amplify/amplify-js/pull/9115 (unit test changes)
- https://github.com/aws-amplify/amplify-js/pull/9251 (this shouldn't be migrated)

Should we do one final migration from JS repo with help from @ErikCH before we merge this PR?

**Comment by ericclemmons** [comment]
@wlee221 Because of the current (and potential future drift), it may be best to update https://github.com/aws-amplify/amplify-ui/tree/legacy/legacy _after_ merge.

Otherwise we may end up repeating this process repeatedly.

**Comment by sammartinez** [comment]
As we have discussed, I am going to close this PR out for now until we move to a breaking change for removal of the UI packages.

**Pull request closed without merging**